pkg/runtime/runtime.go

```diff
@@ -0,0 +1,28 @@
+package runtime
```

Comment:
this feels like an unnecessary layer of abstraction

Reply:
This is fundamentally what the SDK will provide to abstract the dependency on the API server.  It's simply a mechanism by which we can test the code blocks for which we have nothing now and control the outcome of the bits that don't allow us to test now.  It's no different then Mockito as a java library or any other mocking. This is simply an opportunity to add code which will allow us to expand our testing.